python/ray/util/sgd/v2/__init__.py

```diff
@@ -1,4 +1,5 @@
-from ray.util.sgd.v2.backends.torch import TorchConfig
+from ray.util.sgd.v2.backends import BackendConfig, TorchConfig
+from ray.util.sgd.v2.callbacks import SGDCallback
 from ray.util.sgd.v2.trainer import Trainer
 
-__all__ = ["TorchConfig", "Trainer"]
+__all__ = ["BackendConfig", "SGDCallback", "TorchConfig", "Trainer"]
```

Comment:
this naming change was done because Sphinx wouldn't render correctly (conflict with another Callback class in Ray)